Asterix/wfsc/wf_control_functions.py

```diff
@@ -506,7 +513,11 @@ def create_singlewl_interaction_matrix(testbed: Testbed,
                             prop.prop_angular_spectrum(
                                 wavefrontupstreaminDM * (1 + 1j * phase_in_basis) * DM.EF_from_phase_and_ampl(
                                     phase_abb=DM_phase_init[testbed.name_of_DMs.index(DM_name)], wavelengths=wavelength),
-                                wavelength, -DM.z_position, DM.diam_pup_in_m / 2, DM.prad), DM.dim_overpad_pupil)
+                                wavelength,
```

Comment:
Like above with assigning the DM EF to a variable.

Reply:
done